config/tsconfig.base.json

```diff
@@ -7,7 +7,7 @@
         "outDir": "../build",
         "rootDir": "../src",
         "declaration": true,
-        "sourceMap": false,
+        "sourceMap": true,
```

Comment:
Please not in the base config. The `tsconfig.json`configuration does specify `sourceMap: true`. The base should not change that since the production config does not overwrite the setting again.

-> `false`

Reply:
ok